src/components/RecordingDialog.tsx

```diff
@@ -164,25 +167,24 @@ const RecordingDialog = ({
           <ModalBody>
             <VStack width="100%" alignItems="left" gap={5}>
               <VStack height="100px" justifyContent="center">
-                {recordingStatus === RecordingStatus.Recording ? (
-                  <Text
-                    fontSize="5xl"
-                    textAlign="center"
-                    fontWeight="bold"
-                    color="brand.500"
-                  >
+                <Text
+                  fontSize={
+                    recordingStatus === RecordingStatus.Recording
+                      ? "5xl"
+                      : countdownStages[countdownStageIndex].fontSize
+                  }
+                  textAlign="center"
+                  fontWeight="bold"
+                  color="brand.500"
+                  role="region"
```

Comment:
Maybe this is better?
```suggestion
                  role="timer"
```

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/370/commits/0b9dc4b8da1b1a8eff9433ea092bd62e37e88f39